hw-js-7/hw-js-7-3/script.js

```diff
@@ -1,31 +1,105 @@
-  
-  signal.forEach(e => e.classList[e === cs ? 'add' : 'remove'](e.dataset.light));
-
-  timer = setTimeout(init, 5000);
-};
-
-const finish = () => {
-  clearTimeout(timer); 
-  
-  signal.forEach(e => e.classList.remove(e.dataset.light));
-  
-  counter = 0;
-  
-  btnStart.disabled = false;
-  btnStop.disabled = true;
-};
-
-btnStart.addEventListener('click', init);
-btnStop.addEventListener('click', finish);
\ No newline at end of file
+const mainContentEl = document.getElementById("content");
+
+const tableWrapper = document.createElement("div");
+tableWrapper.setAttribute("id","table-wrapper");
+tableWrapper.className = "table-wrapper";
+
+const tableEl = document.createElement("table");
+tableEl.setAttribute("id", "content-table");
+tableEl.className = "content-table";
+
+tableWrapper.append(tableEl);
+mainContentEl.append(tableWrapper);
+
+class Employee {
+  constructor(name, position, department, salary) {
+    this.name = name;
+    this.position = position;
+    this.department = department;
+    this.salary = salary;
+  }
+}
+
+const employees = [
+  new Employee("Vladyslav Vasylenko", "Manager", "Sales", 5000),
+  new Employee("Olga Vasylenko", "DevOps Engineer", "Sales", 1000),
+  new Employee("Vladyslav Sozonov", "Manager", "Sales", 7000),
+  new Employee("Vlady Vasylenko", "Manager", "Sales", 4000),
+  new Employee("Vlado Vasylin", "Manager", "Sales", 3000),
+  new Employee("Vla Asylenko", "Manager", "Sales", 2000),
+  new Employee("Vladoo Vaslo", "Manager", "Sales", 5000),
+  new Employee("Ladyslav Sylenko", "Manager", "Sales", 1000),
+  new Employee("Dyslav VLenko", "Team Lead", "Sales", 2000),
+  new Employee("Vladyslav Vasylenko", "Senior Developer", "Sales", 9000),
+  new Employee("Slava Enko", "Senior Developer", "Sales", 8000),
+  new Employee("Vlev Dasylenko", "Junior Developer", "Sales", 3000),
+  new Employee("Vladyslaven Vasylenkus", "Manager", "Sales", 7000),
+  new Employee("Vladyslav Vasylenko", "Manager", "Sales", 4000),
+  new Employee("Vladyslav Vasylenko", "Junior Developer", "Sales", 5000),
+  new Employee("Vladyslav Vasylenko", "Middle Developer", "Sales", 6000),
+  new Employee("Vladyslav Vasylenko", "Junior Developer", "Sales", 3000),
+  new Employee("Vladyslav Vasylenko", "Junior Developer", "Sales", 4000),
+  new Employee("Vladyslav Vasylenko", "Manager", "Sales", 5000),
+];
+
+class EmpTable {
+  constructor(employees) {
+    this.employees = employees;
+  }
+
+  getHtml() {
+    const table = document.getElementById("content-table");
+    
+    const employees = this.employees;
+
+    const head = document.createElement("tr");
+
+    const th1 = document.createElement("th");
+    th1.textContent = "Name";
+
+    const th2 = document.createElement("th");
+    th2.textContent = "Position";
+
+    const th3 = document.createElement("th");
+    th3.textContent = "Dapartment";
+
+    const th4 = document.createElement("th");
+    th4.textContent = "Salary ($)";
+
+    head.append(th1);
```

Comment:
Same here - a lot of repeating action - `head.append`. When it happens it's time to use functions